lib/dialects/abstract/query-generator.js

```diff
@@ -1702,15 +1702,21 @@ module.exports = (function() {
         })();
       }
 
-      // Do [] to $in/$notIn normalization
+      //Do [] to $in/$notIn normalization
       if (value && (!fieldType || !(fieldType instanceof DataTypes.ARRAY))) {
```

Comment:
Lets keep this block as $in/$notIn normalization and do $not to $ne normalization in another block.